docs/Internal language.md

```diff
@@ -0,0 +1,79 @@
+# The model language
+
+It is important to understand the domain-specific language that describes the data models. It is what the input processors converts their inputs to, and the generators use to generate the output.
```

Comment:
Very personal opinion:
```suggestion
The <whatever name> is what the input processors converts their inputs to, and the generators use to generate the output.
```

I think the first sentence adds nothing useful but confuses the reader.